### ISSUE TITLE
File Limit Request:  aspose-ocr-python-net- 500 MB

### ISSUE THREAD

**Issue by Annushka34** [pypi/support#3250]
### Project URL

https://pypi.org/project/aspose-ocr-python-net/

### Does this project already exist?

- [X] Yes

### New Limit

500

### Update issue title

- [X] I have updated the title.

### Which indexes

PyPI, TestPyPI

### About the project

Information about Aspose.OCR library (Aspose Pty)
https://products.aspose.com/ocr/python-net/
https://releases.aspose.com/ocr/python-net/
we are adding new models and unfortunately exceeded the set limit

### Reasons for the request

Our library uses neural networks, and they have a big weight. They optimized as much as it was possible. These models are the main advantage of our libraries.

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

**Comment by cmaureir** [comment]
Hey @Annushka34 :wave:
Have you explore other ways to distribute your models within your package?
NLTK has a nice approach for people to get the data, for example: https://www.nltk.org/data.html

**Comment by Annushka34** [comment]
Hi, @cmaureir 
Thank you for your reply and suggesting another approach.

Actually, these models are our own development and we intend to distribute our models in one package due to our corporate policies as per our security and safety standards.

**Comment by Annushka34** [comment]
May I kindly request any updates on this matter, please?

**Comment by rgommers** [comment]
> Actually, these models are our own development and we intend to distribute our models in one package due to our corporate policies as per our security and safety standards.

My guess is that this is probably not an acceptable reason. If NLTK and other popular projects like scikit-learn, PyTorch, etc. can distribute data to their large user bases outside of PyPI, there is no real security reason why your proprietary project cannot do the same. PyPI is a space-constrained community resource run largely by volunteers, so "it's corporate policy" unfortunately doesn't work here. 

From a security perspective: if your users already trust a closed source binary from PyPI, they should also be fine trusting a data download from whatever location is specified within that binary.

**Comment by di** [comment]
I've set the upload limit for `aspose-ocr-python-net` to 500MB on PyPI and TestPyPI. Please be mindful of the frequency of releases at that size.
